Scheduler/observing_blocks_maintel/BLOCK-T558.json

```diff
@@ -0,0 +1,128 @@
+{
+    "name": "BLOCK-T558",
+    "program": "BLOCK-T558",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/take_image_lsstcam.py",
+            "standard": true,
+            "parameters": {
+              "nimages": 3,
+              "exp_times": 30,
+              "image_type": "ENGTEST",
+              "reason": "hexapod_compensation_on",
+              "program": "BLOCK-558"
+            }
+          },
+          {
+            "name": "maintel/disable_hexapod_comensation_mode.py",
+            "standard": true   
+          },
+          {
+            "name": "maintel/take_image_lsstcam.py",
+            "standard": true,
+            "parameters": {
+              "nimages": 1,
+              "exp_times": 30,
+              "image_type": "ENGTEST",
+              "reason": "hexapod_compensation_off",
+              "program": "BLOCK-558"
```

Comment:
Here and in every other "compensation_off" script, you need to ignore hexapod_1 and hexapod_2 here otherwise the script will fail waiting for the hexapod to get in position.

```
"ignore": ["mthexapod_1", "mthexapod_2"]
```